etc/helm/pachyderm/templates/pachd/config-secret.yaml

```diff
@@ -0,0 +1,110 @@
+{{- /*
+SPDX-FileCopyrightText: Pachyderm, Inc. <info@pachyderm.com>
+SPDX-License-Identifier: Apache-2.0
+*/ -}}
+{{ if .Values.pachd.enterpriseLicenseKey }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: pachyderm-config
+  namespace: {{ .Release.Namespace }}
+stringData:
+{{ if eq .Values.deployTarget "LOCAL" }}
+  # authConfig is the configuration for pachd as an OIDC client
+  authConfig: |
+    client_id: pachd 
+    client_secret: oidcsecret
```

Comment:
Should probably have a note saying this should match the secret below under `oidcClients`